<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGroupsWindow</class>
 <widget class="QMainWindow" name="AddGroupsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adding Group</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 218, 221)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 236, 236);
font: 63 15pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="GroupsLabel">
        <property name="styleSheet">
         <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
        <property name="text">
         <string>&amp;Group:</string>
        </property>
        <property name="buddy">
         <cstring>GroupsLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="GroupsLineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
        <property name="text">
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="DeleteButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
        <property name="text">
         <string>&amp;Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
